### PR TITLE
iSic Editing 11 June

### PR DESCRIPTION
**Pull request by AlexAAntoniou**
18 Inscriptions

### CODE REVIEW

File: inscriptions/ISic001357.xml [hunk 1]
```diff
@@ -197,58 +202,25 @@
...
-                <ab>
-                    <lb n="1"/>
-
-
-                    ζῶν κὲ <supplied reason="omitted">κ</supplied>είμε-<lb n="2"/>νος ἠγόρασα
-                    <lb n="3"/>
+                    <lb n="1"/><expan><abbr>πρ</abbr><ex>ὸ</ex><abbr>ς</abbr></expan> καλανδ<supplied reason="lost">ῶν</supplied>
+                    <lb n="2"/>Ἰουνίων <unclear>ἐ</unclear><supplied reason="lost">γὼ</supplied>
+                    <lb n="3"/><name>Θέδουλος</name>
+                    <lb n="4"/>ζῶν <orig>ΚΕΕΙΜΕ</orig>
```
Comment (JonPrag) [inline inscriptions/ISic001357.xml:210]: NB Califano interprets thus: 
image

File: inscriptions/ISic001357.xml [hunk 2]
```diff
@@ -197,58 +202,25 @@
...
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI340019">
-                <ab>
-                    <lb n="1"/>
-
-
-                    ζῶν κὲ <supplied reason="omitted">κ</supplied>είμε-<lb n="2"/>νος ἠγόρασα
-                    <lb n="3"/>
+                    <lb n="1"/><expan><abbr>πρ</abbr><ex>ὸ</ex><abbr>ς</abbr></expan> καλανδ<supplied reason="lost">ῶν</supplied>
```
Comment (JonPrag) [inline inscriptions/ISic001357.xml:207]: NB the ς is actually ϛ, i.e. sampi, the numeral 6, not part of the preposition

File: inscriptions/ISic001368.xml [hunk 3]
```diff
@@ -190,37 +195,20 @@
...
-                    <lb n="3"/>ἡ δούλη τοῦ
-                    <supplied reason="lost">θεοῦ</supplied>
-                    <lb n="4"/>Σαβεῖνα τῇ
-                    <supplied reason="omitted">π</supplied>
-                    <supplied reason="lost">ρὸ</supplied>
+                    <lb n="1"/>σὺν θεῷ <expan><abbr><am><g ref="#staurogram">⳨</g></am></abbr><ex>Χριστῷ</ex></expan> <expan><abbr>κ</abbr><ex>αὶ</ex></expan> <supplied reason="lost">πατρι</supplied>
+                    <lb n="2"/>αὐτοῦ ἐκοιμ<supplied reason="lost">ήθη</supplied>
+                    <lb n="3"/>ἡ δούλη τοῦ <supplied reason="lost">θεοῦ</supplied>
+                    <lb n="4"/><name>Σαβεῖνα</name> τῇ <supplied reason="lost">πρὸ</supplied>
                     <lb n="5"/>τρειῶν εἰδῶν
```
Comment (JonPrag) [inline inscriptions/ISic001368.xml:204]: code the numeral, although in words

File: inscriptions/ISic003700.xml [hunk 4]
```diff
@@ -191,73 +196,30 @@
...
+                <listApp>
+                    <app><note>Text: Ferrua 1989</note></app>
+                    <app loc="line 1"><note>IG: [--- Π]α[λλ]αδίου; CIG: [Μνημεῖον Γενν]αδίου</note></app>
+                    <app loc="line 5"><note>IG and CIG: [Σε]π[τ]εμβρίω[ν] </note></app>
+                    <app loc="line 6"><note>CIG: --- ἐμὸν ἃνδρ[α]</note></app>
+                </listApp>
             </div>
             <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+                <p>The purchase of(?) Palladios, the spouse of Pelagia. Pelagia having passed 11 [days] before the Kalends of [...], having died [...] of September, here she, who had one husband, lies</p> <!-- unsure about this translation -->
```
Comment (JonPrag) [inline inscriptions/ISic003700.xml:218]: i think it's fine - agorasia is a standard formulation for the purchase of the loculus /tomb

**Review by JonPrag — CHANGES_REQUESTED** [review]
Bits and pieces, but nothing major!

**Comment by AlexAAntoniou** [comment]
@JonPrag changes now made, and my first attempt at adding metadata for a handful of ILipara inscriptions now added.

**Review by JonPrag — APPROVED** [review]
Dear Alex
this is all excellent, and the Lipara files look very good. We might, perhaps, discuss the question of adding geographic coordinates on the provenance element, but let's have a conversation about that in more detail, perhaps on Friday.

**Pull request merged**